### PR TITLE
DOC: clarification on -b flag in asv

### PR DESCRIPTION
**Pull request by RafalSkolasinski**

### CODE REVIEW

File: doc/source/contributing.rst [hunk 1]
```diff
@@ -525,7 +525,7 @@ supports both python2 and python3.
     so many stylistic issues are likely a result of automated transformation of the
     code.
 
-To install asv::
+To install asv (works also if you are using ``conda``)::
```
Comment (jreback) [inline doc/source/contributing.rst]: this is not necessary

Comment (RafalSkolasinski) [inline doc/source/contributing.rst]: but may be a nice clarification for people not familiar with 'conda' and
'pip'.

Comment (jreback) [inline doc/source/contributing.rst]: no, this will work if you have conda installed or not, so its redundant.

**Comment by jreback** [comment]
thank you sir!

**Pull request merged**